src/addon/calendar/pages/day/day.html

```diff
@@ -68,7 +68,7 @@ <h2><core-format-text [text]="event.name" [contextLevel]="event.contextLevel" [c
 
     <!-- Create a calendar event. -->
     <ion-fab core-fab bottom end *ngIf="canCreate && loaded">
-        <button ion-fab (click)="openEdit()" [attr.aria-label]="'addon.calendar.newevent' | translate">
+        <button ion-fab style="background: #E48E00 !important;" (click)="openEdit()" [attr.aria-label]="'addon.calendar.newevent' | translate">
```

Comment:
Shall we push this css ? @shunmugar06

Reply:
This button colour declared in global theme css file. so that i have added in html inline style class.